sig-advocacy-and-outreach/summer-ospp/README.md

```diff
@@ -10,16 +10,19 @@ KubeSphere in [Summer 2021 of Open Source Promotion Plan](https://summer.iscas.a
+| [Expose LoadBalancer Services Using iptables NAT Rules](expose-loadbalancer-services-using-iptables-nat-rules.md) <br/>Provide a simple mode of [PorterLB](https://porterlb.io/), which allows users to use the IP address of a Kubernetes cluster node and a service port to expose a LoadBalancer service. <br/>Potential Mentor(s): [Zhengyi Lai](https://github.com/zheng1) | Network              | Golang, Kubernetes, Linux network (iptables/IPVS) | Medium | long hong yi |
+| [Support send notifications to pushover](support-send-notifications-to-pushover.md)<br/>Make the notification manager support send notifications to pushover.<br/>Potential Mentor(s):[benjaminhuo](https://github.com/benjaminhuo),[lei](https://github.com/wanjunlei) | Notification Manager | Golang, REST API, Kubernetes                          | Medium | ding zi shuo |
+| [KubeSphere Gatekeeper Integration](kubesphere-gatekeeper-integration.md) <br/>Integrate Gatekeeper as an admission policy engine to KubeSphere. <br/>Potential Mentor(s): [Hongming](https://github.com/wansir/), [Roland](https://github.com/rolandma1986/)| Security | Golang, React, OPA, Gatekeeper, Kubernetes | Medium | tang xian he |
+| [KubeSphere SAML Integration](KubeSphere-SAML-integrations.md) <br/>Integrate SAML authentication as an identity provider to KubeSphere. <br/>Potential Mentor(s): [Hongming](https://github.com/wansir/), [Roland](https://github.com/rolandma1986/)| Security | Golang, REST API/OpenAPI, SAML | Low | None |
+| [OpenFunction's function builder development](OpenFunction-function-builder.md)<br>Develop and optimize function builder to detect Nodejs, Java, PHP or Rust (optional) programs and package functions into application images that can be run in OpenFunction.<br>Potential Mentor(s):[Benjamin Huo](https://github.com/benjaminhuo),[Fang Tian](https://github.com/tpiperatgod/) | Functions-as-a-Service | Golang,Tekton Cloud Native Buildpacks,OpenFunction | Medium | ma peng hui |
+| [OpenFunction's nodejs function framework development](OpenFunction-function-framework-nodejs.md)<br>Develop a functional framework for working with nodejs code, providing users with a suite of tools for their code in a cloud-native environment.<br>Potential Mentor(s):[Benjamin Huo](https://github.com/benjaminhuo),[Fang Tian](https://github.com/tpiperatgod/) | Functions-as-a-Service | Golang,Tekton Cloud Native Buildpacks,OpenFunction,Nodejs | Medium | lin xu ya lun |
+| [OpenFunction website](openfunction-website_zh-CN.md)<br/>在项目导师的帮助下，为全新的 Serverless 开源项目 [OpenFunction](https://github.com/OpenFunction/OpenFunction) 从零开始搭建与开发项目的官方网站，并完成项目上线。<br/>Potential Mentor(s): [Feynman Zhou](https://github.com/FeynmanZhou)、[Sherlock](https://github.com/Sherlock113) | Website | JavaScript，HTML，CSS，Markdown | Low | zhang yuan yi |
```

Comment:
I prefer to only add the English description in this file.

Reply:
Fixed the missing project description in English.

Thanks rick